src/components/Layout/Header/NavBar/MobileNavBar.tsx

```diff
@@ -208,8 +208,8 @@ export const MobileNavBar = memo(() => {
         display={displayProp}
         className='mobile-nav'
       >
-        {allRoutes.map(route => (
-          <MobileNavLink key={route.path} {...route} />
+        {allRoutes.map((route, index) => (
+          <MobileNavLink key={route.path} {...route} order={index < 2 ? index + 1 : index + 2} />
```

Comment:
ensure order is sane with `<Button />` `order` prop below, else things will look... weird.
The reason is yes, we do have priority, but mobile priority is different and the indexes are not the same. Since things are already ordered here with the sorting above, we can leverage said ordering and rely on indexes rather than the wrong priority (not wrong per se, but wrong as it won't be a 1/1 mapping to `order`)

![image](https://github.com/user-attachments/assets/c05d946e-054a-43ca-9ea4-ef127213b15d)